web-app/src/hoc/Layout/container.js

```diff
@@ -10,6 +10,7 @@ export default Wrapped =>
     constructor(props) {
       super(props);
       this.state = {
+        drawerOpen: localStorage.getItem('navDrawerOpen') == 'true' || false,
```

Comment:
why `'true'` instead of `true` here?

I appreciate there may be a good reason for this, but it suggests a code smell to me at first glance

Reply:
localStorage.getItem seems to return true as a string instead of a bool

Reply:
I don't think there's a need for the `|| false` here

Reply:
Also probs best to use the strict comparison `===`

Reply:
@bgribz If localStorage can't find what it's looking for it returns null. I'd rather keep the false fallback to avoid any issues cropping up from accessing null.

Reply:
Would we agree on 
`drawerOpen: localStorage.getItem('navDrawerOpen') === 'true' ? true : false,`

That way there's no string being stored, and there's no null being stored in `drawerOpen`.

Reply:
But you wouldn't be setting `null`, you'd be setting the result of `null === 'true' => false`

Reply:
gotcha - my head isn't in the game this morning 😄